tests/net/all/prj.conf

```diff
@@ -106,8 +106,6 @@ CONFIG_NET_CONTEXT_SYNC_RECV=y
 CONFIG_NET_CONTEXT_CHECK=y
 
 # SLIP
-CONFIG_NET_SLIP_TUN=n
-CONFIG_NET_SLIP_TAP=y
```

Comment:
Or actually we could try to keep the CONFIG_NET_SLIP_TAP=y option as the TUN option is now removed.